egs/librispeech/ASR/zipformer/onnx_pretrained-streaming.py

```diff
@@ -19,7 +19,7 @@
 repo=$(basename $repo_url)
 
 pushd $repo
-git lfs pull --include "data/lang_bpe_500/bpe.model"
+git lfs pull --include "data/lang_bpe_500/tokens.txt"
```

Comment:
```suggestion
```

Reply:
`git lfs` commands for pulling `tokens.txt` have been removed in the latest commit.